From: Olivier Tilloy <olivier@tilloy.net>
Date: Thu, 18 Dec 2025 14:27:34 +0200
Subject: Strip binaries when building Spidermonkey, to save disk space
Forwarded: not-needed

--- a/libraries/source/spidermonkey/build.sh
+++ b/libraries/source/spidermonkey/build.sh
@@ -171,7 +171,8 @@
 	# shellcheck disable=SC2086
 	MOZCONFIG="$(pwd)/../mozconfig" \
 	MOZCONFIG_OPTIONS="${CONF_OPTS} \
-		--enable-optimize" \
+		--enable-optimize \
+		--enable-strip" \
 	BUILD_DIR="build-release" \
 		./mach build ${JOBS}
 )

