From: Olivier Tilloy <olivier@tilloy.net>
Date: Fri, 19 Dec 2025 22:31:11 +0200
Subject: Do not unpack unneeded files when building Spidermonkey, to save disk space
Forwarded: not-needed

--- a/libraries/source/spidermonkey/build.sh
+++ b/libraries/source/spidermonkey/build.sh
@@ -69,7 +69,18 @@
 
 # unpack
 rm -Rf "${FOLDER}"
-"${TAR}" xfJ "${FOLDER}.tar.xz"
+"${TAR}" xfJ "${FOLDER}.tar.xz" \
+	--exclude=js/src/tests/non262 \
+	--exclude=js/src/tests/test262 \
+	--exclude=js/src/jit-test \
+	--exclude=python/mozperftest \
+	--exclude=testing/web-platform \
+	--exclude=third_party/rust/winapi*/lib/*.a \
+	--exclude=third_party/rust/winapi/src/shared \
+	--exclude=third_party/rust/winapi/src/um \
+	--exclude=third_party/rust/windows-sys/src/Windows \
+	--exclude=third_party/rust/mp4parse/link-u-avif-sample-images
+rm -Rf "${FOLDER}.tar.xz"
 
 # patch
 (

