From: Olivier Tilloy <olivier@tilloy.net>
Date: Mon, 13 Oct 2025 19:23:42 +0200
Subject: Do not build Spidermonkey in debug mode, to save time and disk space
Forwarded: not-needed

--- a/libraries/source/spidermonkey/build.sh
+++ b/libraries/source/spidermonkey/build.sh
@@ -129,7 +129,7 @@
 
 	# Build
 	# Debug (broken on FreeBSD)
-	if [ "${OS}" != "FreeBSD" ]; then
+	if false; then
 		MOZCONFIG="$(pwd)/../mozconfig" \
 		MOZCONFIG_OPTIONS="${CONF_OPTS} \
 			--enable-debug \
@@ -190,7 +190,7 @@
 mkdir include-release
 cp -R -L "${FOLDER}"/build-release/dist/include/* include-release/
 
-if [ "${OS}" != "FreeBSD" ]; then
+if false; then
 	mkdir include-debug
 	cp -R -L "${FOLDER}"/build-debug/dist/include/* include-debug/
 fi
@@ -222,7 +222,7 @@
 else
 	# Copy shared libs to lib/, they will also be copied to binaries/system, so the compiler and executable (resp.) can find them.
 	cp -L "${FOLDER}/build-${REL}/js/src/build/${LIB_PREFIX}${LIB_NAME}-${REL}${LIB_SUFFIX}" "lib/${LIB_PREFIX}${LIB_NAME}-${REL}${LIB_SUFFIX}"
-	if [ "${OS}" != "FreeBSD" ]; then
+	if false; then
 		cp -L "${FOLDER}/build-${DEB}/js/src/build/${LIB_PREFIX}${LIB_NAME}-${DEB}${LIB_SUFFIX}" "lib/${LIB_PREFIX}${LIB_NAME}-${DEB}${LIB_SUFFIX}"
 	fi
 fi
